cmd/web.go

```diff
@@ -190,10 +205,6 @@ func setPort(port string) error {
 		defaultLocalURL += ":" + setting.HTTPPort + "/"
 
 		cfg.Section("server").Key("LOCAL_ROOT_URL").SetValue(defaultLocalURL)
-
-		if err := cfg.SaveTo(setting.CustomConf); err != nil {
-			return fmt.Errorf("Error saving generated JWT Secret to custom config: %v", err)
-		}
```

Comment:
--port states that it is temporary so it shouldn't write to the ini... 

however, this is going to break gitea serv, admin, manager etc. I just don't know what to do here.

In terms of doing a pure bugfix for the linked Issue only this change should be dropped but it's really hard to know what to do here.